apps/pages/templates/pages/review_guide.html

```diff
@@ -83,7 +80,7 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit one-word reviews such as "Great!", "wonderful," or "bad."
+          Submit one-word reviews such as "Great!", "Wonderful", or "Bad".
```

Comment:
I wouldn't want your head to explode. What if we ditched the list and just said, "Submit one-word reviews."